### DISCUSSION TITLE
Erweiterung des PCB um MPPT & (Pylontech) CAN BUS Schnittstellen

### DISCUSSION THREAD

**Discussion by swingstate** [dokuhn/openDTU-BreakoutBoard#22] [Ideas]
Hallo, 

food for thought - eine Erweiterung des PCBs um [dieses Project](https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/128) näher in Richtung Plug and Play zu bringen wäre cool. 

Vielleicht ist das etwas für eine Version 3? :)

**Comment by thomic** [comment]
Moin,
ich denke als Projekt muss man seinen Scope kennen und den Aufwand den man als freiwillige Arbeit leisten kann.
Das hier ist GitHub und ein open hardware project, es steht jedem zu forken und darauf basierende Platinen zu entwickeln.
Für dieses Repo, ist das glaub ich eher keine Richtung in die wir uns entwickeln wollen, um die Komplexität überschaubar zu halten.
